app/code/Magento/AsynchronousOperations/Model/ResourceModel/Operation/OperationRepository.php

```diff
@@ -91,7 +91,7 @@ public function createByTopic($topicName, $entityParams, $groupId, $operationId)
         ];
         $data = [
             'data' => [
-                OperationInterface::ID => $operationId,
+                OperationInterface::OPERATION_KEY => $operationId,
```

Comment:
It should not work correctly.
OPERATION_KEY - its a DB key for "operation_key" column.

if you switch OPERATION_KEY to "id", it will stop working, cause $operationId here is a key of array, and in database "id" is an auto_increment value. And in case of multiple operations, will be duplicated for different bulk operations.

Implementing of OPERATION_KEY was done in purpose, for decouple operation keys from database increments and improve performance on saving big operations.

Another issue, it that not all Magento API functionality have a tests coverage, so we overslept broken another part of API